### PR TITLE
Users/balaga gayatri/multicontainer support

### PR DESCRIPTION
**Pull request by BALAGA-GAYATRI**
Added changes to accept compose file for multicontainer apps along with images

### CODE REVIEW

File: src/DeploymentProvider/DeploymentProviderFactory.ts [hunk 1]
```diff
@@ -18,8 +18,13 @@ export class DeploymentProviderFactory {
             }
         }
         else if(type == DEPLOYMENT_PROVIDER_TYPES.SPN) {
-            if (!!ActionParameters.getActionParams().images) {
-                return new WebAppContainerDeploymentProvider(type);
+            if(!!ActionParameters.getActionParams().isLinux){
+                if (!!ActionParameters.getActionParams().images || !!ActionParameters.getActionParams().multiContainerConfigFile) {
```
Comment (AmrutaKawade) [inline src/DeploymentProvider/DeploymentProviderFactory.ts:22]: you can combine these three condition in single if condition

File: src/ActionInputValidator/ValidatorFactory.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export class ValidatorFactory {
             appNameIsRequired(actionParams.appName);
             await this.getResourceDetails(actionParams);
             if (!!actionParams.isLinux) {
-                if (!!actionParams.images) {
+                if (!!actionParams.images || !!actionParams.multiContainerConfigFile) {
```
Comment (AmrutaKawade) [inline src/ActionInputValidator/ValidatorFactory.ts:33]: need to add same condition on line 41

File: lib/actionparameters.js [hunk 3]
```diff
@@ -1,103 +1,116 @@
...
-    }
-    get multiContainerConfigFile() {
-        return this._multiContainerConfigFile;
-    }
-}
-exports.ActionParameters = ActionParameters;
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
```
Comment (AmrutaKawade) [inline lib/actionparameters.js:4]: what are these changes?

Comment (BALAGA-GAYATRI) [reply lib/actionparameters.js:4]: Don't know I ran `npm run build ` to my changes and these came. Not sure from where they came.

**Pull request closed without merging**